tests/e2e/backup_restore_suite_test.go

```diff
@@ -235,10 +262,6 @@ var _ = Describe("AWS backup restore tests", func() {
 			restoreName := fmt.Sprintf("%s-%s", brCase.Name, restoreUid.String())
 
 			// install app
-			updateLastInstallTime()
-			log.Printf("Installing application for case %s", brCase.Name)
-			err = InstallApplication(dpaCR.Client, brCase.ApplicationTemplate)
-			Expect(err).ToNot(HaveOccurred())
```

Comment:
This was a duplicate call to install application causing majority of the issues.